Makefile

```diff
@@ -187,7 +190,21 @@ docker: _print_vars docker-build docker-push
 
 ### docker-build: Builds the controller image
 docker-build:
-	$(DOCKER) build . -t ${DWO_IMG} -f build/Dockerfile
+	if [ "$(ARCH)" = "arm64" ]; then \
+		# build natively only the first stage
+		$(DOCKER) build . --target builder -t devworkspace-builder:temp -f build/Dockerfile; \
+		# create a container from the temporary image
+		$(DOCKER) create --name temp-builder devworkspace-builder:temp; \
+		# copy the binaries to a new local directory
+		$(DOCKER) cp temp-builder:/devworkspace-operator/_output/bin ./_output; \
+		# remove the temporary container
+		$(DOCKER) rm temp-builder; \
+		$(DOCKER) build . --platform linux/amd64 -t ${DWO_IMG} -f build/Dockerfile.arm64; \
+		# remove the temporary directory
+		#rm -rf ./_output; \
+	else \
+		$(DOCKER) build . -t ${DWO_IMG} -f build/Dockerfile; \
+	fi
```

Comment:
@akurinnoy I don't fully understand, could you please explain why need a separate dockerfile for arm64? Are there problems when running `compile-devworkspace-controller` and `compile-webhook-server` on arm64?

Reply:
The problem is that reusing the original multi-stage file for the final build forces re-run of the `builder` stage under emulation, which causes the Go compiler to crash. The separate `Dockerfile.arm64` breaks this dependency by copying the pre-built binaries from local disk without triggering the re-build.